CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Version 2.1.0 - New feature release - 2024-12
+### Resampling recipe
+- :date: Support selecting custom dates for the start and end of extrapolation
+
```

Comment:
I'll just revert that change, I bumped pandas without thinking because tests failed to run with pytest latest but it's not needed in the end (we keep pytest to less than 8)

Reply:
Wait it's only for unit tests, that's why I did it without too much care. Do we want to mention that in the changelog still?

Reply:
oh my bad 🤦 in that case, no need to mention i don't think